axelrod/tests/unit/test_human.py

```diff
@@ -91,3 +91,7 @@ def test_strategy(self):
         expected_action = C
         actual_action = human.strategy(Player(), lambda: C)
         self.assertEqual(actual_action, expected_action)
+
+    def test_reset_history_and_attributes(self):
```

Comment:
Is this necessary? I though there was a catch in TestPlayer for this strategy. If you've removed that then ignore.